client/src/Components/ReviewsChart.jsx

```diff
@@ -8,20 +9,104 @@ import './../styles/ReviewsChart.scss';
 //   |____|_  /\___  >\_/ |__|\___  >\/\_//____  >\______  /___|  (____  /__|   |__|  
 //          \/     \/             \/           \/        \/     \/     \/             
 
-const ReviewsChart = (props) => {
```

Comment:
I believe Airbnb dictates that a single argument doesn't need parens

Reply:
This line was actually removed in this changeset.